pyEpiabm/pyEpiabm/tests/testing_parameters.json

```diff
@@ -461,5 +461,32 @@
                                     0.47651, 0.47651, 0.47651, 0.47651,
                                     0.47651, 0.47651, 0.47651, 0.47651,
                                     0.47651]
+    },
+    "household_age_params": {
```

Comment:
The interpretation of all these parameters (and their sources) need to be included in the wiki - see Patricia's style for examples.